source/ui/BioXAS/BioXASSideXASScanConfigurationView.cpp

```diff
@@ -255,3 +262,9 @@ void BioXASSideXASScanConfigurationView::onEdgeChanged()
 	if (energy_->value() != configuration_->energy())
 		energy_->setValue(configuration_->energy());
 }
+
+void BioXASSideXASScanConfigurationView::onEstimatedTimeChanged(double time)
+{
+    estimatedTimeLabel_->setText(time);
```

Comment:
You'll still need the AMDateTimeUtils class to transform the double into a proper time.  Also, I don't think this would compile.